pavelib/tests.py

```diff
@@ -212,6 +212,26 @@ def coverage(options):
     call_task('diff_coverage', options=dict(options))
 
 
+@no_help
+@task
+@needs('pavelib.prereqs.install_prereqs')
+def combine_jenkins_coverage():
+    """
+    Combine coverage reports.
+    """
+    coveragerc = Env.REPO_ROOT / 'test_root' / '.jenkins-coveragerc'
+
+    for directory in Env.LIB_TEST_DIRS + ['cms', 'lms']:
+        report_dir = Env.REPORT_DIR / directory
+        if report_dir.isdir():
```

Comment:
When would this be false and why would you do nothing in this case? Include a comment above this line explaining why the if statement is needed.